gp-includes/routes/note.php

```diff
@@ -96,8 +96,8 @@ public function delete_post() {
 		}
 
 		$this->notices[] = __( 'The note was deleted!', 'glotpress' );
-		GP::$notes->delete_all( array( 'id' => $note_id ) );
+		GP::$notes->id   = $note_id;
```

Comment:
Why not `$note->delete()`? `$this->notices[]` should only be added if `$note->delete()` was successful.

Reply:
so you mean that is better to put the notice with a check in this method or put that var inside the delete method itself?